src/internal/NeoEsp32RmtMethod.h

```diff
@@ -554,7 +554,7 @@ template<typename T_SPEED, typename T_CHANNEL> class NeoEsp32RmtMethodBase
         config.clk_div = T_SPEED::RmtClockDivider;
 
         ESP_ERROR_CHECK(rmt_config(&config));
-        ESP_ERROR_CHECK(rmt_driver_install(_channel.RmtChannelNumber, 0, ESP_INTR_FLAG_IRAM | ESP_INTR_FLAG_LEVEL1));
+        ESP_ERROR_CHECK(rmt_driver_install(_channel.RmtChannelNumber, 0, 0));

```

Comment:
I believe the "correct" change is to use `ESP_INTR_FLAG_LOWMED` rather than `0`

Reply:
I will try with `ESP_INTR_FLAG_LOWMED`.
From the tests I have done on ESP32 and ESP32C3, it's super stable with `0`, no more crashes (idf 4.4)

Reply:
The crash does not appear with `ESP_INTR_FLAG_LOWMED`, let me change the PR